packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -336,4 +344,9 @@ export enum StreamEncryption {
    * If `encryptionKey` is specified, this key will be used, otherwise, one will be defined.
    */
   KMS = 'KMS',
+
+  /**
+   * Server-side encryption with a master key managed by Amazon Kinesis
+   */
+  KINESIS_MANAGED = 'MANAGED'
```

Comment:
Just `MANAGED` should suffice?

Reply:
was trying to be more consistent with how we name them elsewhere, but it seems `KMS_MANAGED` is how we've named it for SQS [Queues](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-sqs.QueueEncryption.html) and I find that confusing when I compare it to S3 [bucket encryption](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.BucketEncryption.html)

I'm okay with `MANAGED` as it communicates the intent just as well.